kubernetes/prod/configmap.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: moderator-prod
   namespace: moderator-prod
 data:
-  ALLOWED_HOSTS: "moderator-prod.itsre-apps.mozit.cloud"
```

Comment:
I would keep both hosts, separated by a comma

Reply:
I like the idea too, but I wasn't sure about the formatting. Fixed now